src/pages/components/color-swatch.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Color swatch
+description: >
+  The Color swatch allows users to choose colors using sliders and input fields.
+package: '@zendeskgarden/react-colorpickers'
+components:
+  - colorpickers/src/elements/ColorSwatch/index.tsx
+  - colorpickers/src/elements/ColorSwatchDialog/index.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To let users change, update, or add predefined colors</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          For more finite control over color selection, use the [Color
+          picker](/components/color-picker)
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The Color swatch allows users to select a single predefined color.
+
+import ColorSwatchDefault from '../../examples/color-swatch/ColorSwatchDefault.tsx';
+import ColorSwatchDefaultCode from '!!raw-loader!../../examples/color-swatch/ColorSwatchDefault.tsx';
+
+<CodeExample code={ColorSwatchDefaultCode}>
+  <ColorSwatchDefault />
+</CodeExample>
+
+### Custom trigger
+
+The Color swatch can be displayed using a trigger, e.g. an [Icon button](/components/icon-button).
+
+import ColorSwatchCustomTrigger from '../../examples/color-swatch/ColorSwatchCustomTrigger.tsx';
+import ColorSwatchCustomTriggerCode from '!!raw-loader!../../examples/color-swatch/ColorSwatchCustomTrigger.tsx';
+
+<CodeExample code={ColorSwatchCustomTriggerCode}>
+  <ColorSwatchCustomTrigger />
+</CodeExample>
+
+### Dialog
+
+The Color swatch can be placed inside a dialog.
```

Comment:
@james-sann 

Technically, consumers cannot place the color swatch in the dialog. It comes as a stand alone component called `<ColorSwatchDialog>`.

However, consumers can place the `<ColorSwatch>` in their own custom dialog implementations:

```jsx
<CustomDialog>
  <ColorSwatch />
  <Button>Toggle</Button>
</CustomDialog>
```

Reply:
However, this is the same description used on the `ColorPicker` page (for a parallel component).

Reply:
@jzempel Does it make sense to adjust both the `ColorPicker` and `ColorSwatch` **Dialog** section page wordings? ccing @diana-oum

Reply:
I'm all for improving the descriptions across the board.

Reply:
Thanks for the tag James.
I think the nuance that Henry brought up is important. I personally didn't receive that nuance from how the description is worded today.

How about "The Color swatch can be shown inside a dialog using ColorSwatchDialog."? That way we get rid of the "placed" verb that conveys different expectations.

Reply:
Can this be updated in the Figma asset first? That way the code can ensure matching style with the Figma asset.

Reply:
Update: I'm going to merge this as-is since @jzempel pointed out the same (somewhat inaccurate) description is used for the color picker. I'll create a _new_ task update the copy. Would still be great to have that reflected on the Figma assets.

Reply:
@hzhu [Color Swatch Figma document copy updated](https://www.figma.com/file/TyNwV6sLH0K04JlXeMWpRt/Color-Swatch?node-id=48%3A2874).